src/Umbraco.Web.UI.Client/src/views/content/overlays/save.controller.js

```diff
@@ -58,57 +58,47 @@
         function onInit() {
             vm.variants = $scope.model.variants;
             vm.availableVariants = vm.variants.filter(saveableVariantFilter);
+            vm.isNew = vm.variants.some(variant => variant.state === 'NotCreated');
 
-            if(!$scope.model.title) {
-                localizationService.localize("content_readyToSave").then(function(value){
+            if (!$scope.model.title) {
+                localizationService.localize("content_readyToSave").then(value => {
                     $scope.model.title = value;
                 });
             }
 
-            _.each(vm.variants,
-                function (variant) {
+            vm.variants.forEach(variant => {
 
-                    //reset state:
-                    variant.save = false;
-                    variant.publish = false;
+                //reset state:
+                variant.save = variant.publish = false;
+                variant.isMandatory = isMandatoryFilter(variant);
 
-                    variant.isMandatory = isMandatoryFilter(variant);
-
-                    if(variant.state !== "NotCreated"){
-                        vm.isNew = false;
-                    }
-                });
-
-            _.each(vm.variants,
-                function (variant) {
-                    if(vm.isNew && hasAnyData(variant)){
-                        variant.save = true;
-                    }
-                });
+                if(vm.isNew && hasAnyData(variant)){
+                    variant.save = true;
+                }
+            });
 
             if (vm.variants.length !== 0) {
-
-                _.find(vm.variants, function (v) {
-                    if(v.active) {
-                        //ensure that the current one is selected
-                        v.save = true;
-                    }
-                });
-
-                vm.availableVariants.sort(function (a, b) {
+                        
+                //ensure that the current one is selected
+                var active = vm.variants.find(v => v.active);
+                if (active) {
+                    active.save = true;
+                }
+
+                vm.availableVariants.sort((a, b) => {
                     if (a.language && b.language) {
-                        if (a.language.name > b.language.name) {
+                        if (a.language.name < b.language.name) {
                             return -1;
                         }
-                        if (a.language.name < b.language.name) {
+                        if (a.language.name > b.language.name) {
                             return 1;
                         }
```

Comment:
I'm seeing this reversal a few times @nathanwoulfe - that's scary! Can you explain? I don't get why it's the other way around now..